jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -170,7 +170,11 @@
 		
 		/* single cached value of public Lookup object */
 		/*[IF Sidecar19-SE-OpenJ9]
+		/*[IF Java14]*/
+		static Lookup PUBLIC_LOOKUP = new Lookup(Object.class, Lookup.UNCONDITIONAL);
+		/*[ELSE]*/
 		static Lookup PUBLIC_LOOKUP = new Lookup(Object.class, Lookup.PUBLIC | Lookup.UNCONDITIONAL);
+		/*[ENDIF] Java14 */
 		/*[ELSE]*/
 		static Lookup PUBLIC_LOOKUP = new Lookup(Object.class, Lookup.PUBLIC);
```

Comment:
Sincere apology for late response, somehow I didn't catch the git notification (and strangely it didn't show up with mention query in the past).
I think `JPP` tags could be organized as following: 
```suggestion
		static final int mhMask = 
		/*[IF Java11]
		/*[IF Java14]*/
		Lookup.UNCONDITIONAL;
		/*[ELSE]*/
		Lookup.PUBLIC | Lookup.UNCONDITIONAL;
		/*[ENDIF] Java14 */
		/*[ELSE] Java11 */
		Lookup.PUBLIC;
		/*[ENDIF] Java11 */
		static Lookup PUBLIC_LOOKUP = new Lookup(Object.class, mhMask);
```
Alternatively `static final int mhMask = ` could be moved within the `JPP` decoration to avoid the actual code like 
```
		static final int mhMask = 
		Lookup.PUBLIC;
```

Reply:
I will update the code against the suggestion above by @JasonFengJ9 if there is no other concern on the format.

Reply:
The code there has been updated as suggested above.